package.json

```diff
@@ -52,7 +53,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
-    "popsicle": "^8.2.0",
+    "any-promise": "^1.3.0",
```

Comment:
I've already changed this, it's all good. In general I prefer `any-promise`. It's overhead is minimal, even on the browser, and most of my projects have support `<= 0.10`. Some people, including me, don't like to pollute the global for whatever reason.